apis/otpapi/otpapi.yaml

```diff
@@ -0,0 +1,407 @@
+openapi: 3.0.1
+info:
+  title: OTP API(s)
+  description: 'The Email resources provides the notification mechanism for all the
```

Comment:
@sknirmalkar89 this microservice description is a copy-paste of the notification description. Please correct it based on the comments given for the previous API.

Reply:
updated